doc-content/drools-docs/src/main/asciidoc/DecisionEngine/phreak-sequential-mode-con.adoc

```diff
@@ -0,0 +1,85 @@
+[id='phreak-sequential-mode-con_{context}']
+
+= Sequential mode in Phreak
+
+Sequential mode is a rule base configuration in the {DECISION_ENGINE}, supported by Phreak, that enables rules in a stateless KIE session to access the data modified by the `modify` or `update` statements of previous rules. As a result, rule execution is faster in sequential mode because the data does not change after the initial data set insertion. Rules can be influenced by the execution of other rules for a more intuitive and holistic rule evaluation. Sequential mode also disables inference in rule evaluation.
```

Comment:
I don't understand what you mean with "holistic rule evaluation" here. I'd remove that sentence.

Reply:
Done. However, can you reply to [BXMSDOC-3759](https://issues.jboss.org/browse/BXMSDOC-3759) about this? The description here and in the Rule base configuration section about sequential mode is unclear and confusing in terms of inference, etc. Toshiya brought this up and I've pinged you, Luca, and Edoardo there.